testnets/symphonytestnet3/assetlist.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "symphonytestnet",
+  "assets": [
+    {
+      "description": "The native token of Symphony",
+      "denom_units": [
+        {
+          "denom": "note",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "mld",
+          "exponent": 6,
+          "aliases": [
+            "melody"
+          ]
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "note",
+      "name": "Melody",
+      "display": "mld",
+      "symbol": "MLD",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/symphonytestnet/images/mld.png"
```

Comment:
These URIs are fine as they are (pointing to the /sympthonytestnet/images/ directory), but the clones uploads are not required and are serving no purpose.